packages/app-desktop/jest.config.js

```diff
@@ -128,7 +128,7 @@ module.exports = {
 	testEnvironment: 'jsdom',
 
 	// Options that will be passed to the testEnvironment
-	// testEnvironmentOptions: {},
+	testEnvironmentOptions: { resources: 'usable' },
```

Comment:
why?

Reply:
I found out that jest is not loading images when src is set and because of this onload func is never called which is used by ‘svgUriToPng’ function. On some research I found this solution to load resources while testing. 
reference:  https://stackoverflow.com/a/49482563

Reply:
Oh thanks for clarifying. Could you please add a comment there with a link to the SO question?